raiden/transfer/mediated_transfer/initiator.py

```diff
@@ -370,7 +374,11 @@ def handle_secretrequest(
 
     elif not is_valid_secretrequest and is_message_from_target:
         initiator_state.received_secret_request = True
-        iteration = TransitionResult(initiator_state, list())
+        invalid_request = EventInvalidSecretRequest(
+            intended_amount=initiator_state.transfer_description.amount,
+            actual_amount=state_change.amount,
+        )
+        iteration = TransitionResult(initiator_state, [invalid_request])
```

Comment:
Why is this variable called `iteration`?  Is there something going over something again and again?

Reply:
I think it relates to the iteration of the state machine?

Reply:
I'll ask further on the knowledge base.

Reply:
Asked https://ethereum.stackexchange.com/questions/74797/in-raidens-source-why-is-a-result-of-state-transition-called-iteration .